.gitignore

```diff
@@ -20,6 +20,7 @@ doc/litepaper/main.pdf
 # Python
 *.pyc
 *.egg-info
+src/python/sdk/docs/*
```

Comment:
Prefer per-project gitignores. This repo contains a lot of different languages, all with different files that should be ignored.

.gitignore files in subdirectories also prevents issues from arising when directory structure changes

Reply:
@bogberry addressed in bea4ab5

See also https://linear.app/econia-labs/issue/ECO-281/enforce-granular-gitignores